common/src/main/java/org/apache/rocketmq/common/MixAll.java

```diff
@@ -77,6 +77,7 @@ public class MixAll {
     public static final long FIRST_SLAVE_ID = 1L;
     public static final long CURRENT_JVM_PID = getPID();
     public final static int UNIT_PRE_SIZE_FOR_MSG = 28;
+    public final static int ALL_ACK_IN_SYNC_STATE_SET_NUM = -1;
```

Comment:
Would better just name it in ALL_ACK_IN_SYNC_STATE_SET without NUM since it's only a constant status. Adding NUM seems a bit weird.